src/actions/browser_action.cr

```diff
@@ -27,4 +33,32 @@ abstract class BrowserAction < Lucky::Action
   private def find_current_user(id) : User?
     UserQuery.new.id(id).first?
   end
+
+  def store_breeze_request : Continue
+    req = BreezeRequest::SaveOperation.create!(
```

Comment:
Will slow down dev env ever so slightly, but prob worth it. Just make sure to have a Habitat option to disable breeze